### DISCUSSION TITLE
ERROR: File is not a zip file with Zipfile - please help!!

### DISCUSSION THREAD

**Discussion by naturallyherb** [kubilus1/autoortho#278] [General]
This problem has been popping up within the last week when a new update to North America became available (0.0.54). It's displaying this error every time I download this update, and because the new update erases previous ortho files, this means that Auto Ortho is completely unusable.

It appears that while z_na_00.zip was generated in the first place, it is completely deleted when the error appears (see screenshots below).

I am using Windows 10 and X-Plane 11.

I am deeply disappointed and extremely frustrated with the developers ever since this problem surfaced and that no action appears to be taken to rectify this issue. Please help! Or else revert this whole thing back to an earlier version from like May 2023 where it was working perfectly fine!

Zip file before error
![!ao problem 1](https://github.com/kubilus1/autoortho/assets/113274769/405c220f-097d-449a-af27-584c8484fb28)

Zip file after error
![!ao problem 3](https://github.com/kubilus1/autoortho/assets/113274769/7b85ea12-3362-4a6f-bda5-a1393e57091c)

The error itself
![!ao problem 2](https://github.com/kubilus1/autoortho/assets/113274769/06d974e2-662c-4e18-bec7-b2f74e231e53)

Log file
[autoortho.log](https://github.com/kubilus1/autoortho/files/12091764/autoortho.log)

**Comment by rayw45** [comment]
Just a thought. Are you using WinRAR on your system for decompressing files?? Are you sure it is a .zip file and not a .rar??   Maybe a silly suggestion but an easy oversight to make.

**Comment by dpdurst** [comment]
Are you using the 4.4 version of the 5.x version of the program? I've had no problem at all with the 5.x program and unzipping the NA build as soon a it came out. Maybe try the other version and see if you have the same thing. Also you can save the zips from being deleted by changing the flag below to false in the auto ortho ini. 

don't cleanup downloads
noclean = False


You do have plenty of space to not only DL but unzip as well on the same drive? You sure its not something with the drive? I don't Winrar being the issue I use it and have no issues unzipping them.

have you tried an earlier version of the NA builds?
https://github.com/kubilus1/autoortho-scenery/releases

**Comment by rayw45** [comment]
Sadly as I use primarily Asia and Australia packs I am stuck still with
versions 44-1 and 42-1 respectively.  Does anyone know when these will be
updated too??  What advantages will it give me when it does update???

On Wed, 19 Jul 2023 at 17:27, dpdurst ***@***.***> wrote:

> Are you using the 4.4 version of the 5.x version of the program? I've had
> no problem at all with the 5.x program and unzipping the NA build as soon a
> it came out. Maybe try the other version and see if you have the same
> thing. Also you can save the zips from being deleted by changing the flag
> below to false in the auto ortho ini.
>
> You do have plenty of space to not only DL but unzip as well on the same
> drive? You sure its not something with the drive? I don't Winrar being the
> issue I use it and have no issues unzipping them.
>
> have you tried an earlier version of the NA builds?
> https://github.com/kubilus1/autoortho-scenery/releases
>
> [scenery]
> don't cleanup downloads
>
> noclean = False
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/kubilus1/autoortho/discussions/278#discussioncomment-6487328>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABEXNKTHFKZ7INMUZTWC2OLXQ6ZCVANCNFSM6AAAAAA2PNQYWI>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>
-- 
Best Regards
Ray Walton
Sent from my iPhone

**Comment by huahezu** [comment]
I uninstalled winrar and installed 7 zip and correctly downloaded all of South America. It just got stuck downloading https://github.com/kubilus1/autoortho-scenery/releases/download/0.0.53/y_sa_overlays.zip.0 but I closed the app, reopened it and started the install and it started where I left off and everything (finally) has been installed correctly.

**Comment by stevepurton** [comment]
I have same problem as naturallyherb how do I try an earlier version, do I have to download separately all the individual zip  files ??

**Reply by stevepurton** [reply]
Just updated South America and got the same error problem, as a result I have now lost both North and South America sceneries, totally lost faith with this product and the author does not seem to respond to problems ??

**Comment by Meteoroid100** [comment]
@naturallyherb Hey there!  Have you found a solution to this yet?  I am having the same issue, and it is really annoying.  I would love to figure this out!  Thanks!

**Comment by stevepurton** [comment]
Hi
Sadly no, thought I had cracked it with a solution I found on the x plane org forum but no luck.
Must say I am disappointed that kubilus does not seem to respond to messages about this problem, you and I don't seem to be the only ones with this problem.
Keep the blue side up !!
Steve


Sent from Outlook<http://aka.ms/weboutlook>

________________________________

**Reply by Meteoroid100** [reply]
Thanks for the response!  That's too bad a solution hasn't been found yet.  Keep me posted :)

**Comment by macmaccas** [comment]
I had the same problem I run it as Administrator and it worked fine.

**Comment by stevepurton** [comment]
Thanks for your response, sadly did not work for me still getting same error message.


Sent from Outlook<http://aka.ms/weboutlook>

________________________________

**Comment by stevepurton** [comment]
Hi
I have now successfully updated North and South America.
I don't know if the actions I took solved the problem or if the program just decided to run OK but here is what I did..................
I had the Image Cache Directory on my C drive and I noticed that during updating the drive was nearly full so I moved the directory to another drive which had much more free space.
Turned off my firewall
Ran the program as an administrator.
Both North and South America updated without any problems.
Regards
Steve



Sent from Outlook<http://aka.ms/weboutlook>

________________________________

**Comment by macmaccas** [comment]
Nice to hear great news!

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

**Reply by janjacobs66** [reply]
Afraid that did not work for me, my cache drive is on 15% full, stopped the FW & AV, I always run it as Administrator anyway

**Comment by janjacobs66** [comment]
Same issue, followed the advice ..
I deleted all in the downloads folder ( in user), nothing partially downloaded in scenery folder (x-plane)
Still can't complete NA or SA - get same error every time
"ERROR File is not a zip with Zipfile ....... .autoortho-data\downloads"
always z_na_00.zip & always z_sa_00.zip
